openshift/4.0/templates/Logging/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM frolvlad/alpine-glibc:latest
+
+ARG OC_VERSION=3.11.173
+ARG BUILD_DEPS='tar gzip'
+ARG RUN_DEPS='bash curl ca-certificates gettext'
+
+ARG APP_ROOT=/opt/app
+ARG APP_LOG=/logging/
+
+RUN apk update && apk upgrade && apk add openssh && apk add zip && apk --no-cache add $BUILD_DEPS $RUN_DEPS && \
+    curl -sLo /tmp/oc.tar.gz https://mirror.openshift.com/pub/openshift-v$(echo $OC_VERSION | cut -d'.' -f 1)/clients/$OC_VERSION/linux/oc.tar.gz && \
+    tar xzvf /tmp/oc.tar.gz -C /usr/local/bin/ && \
+    rm -rf /tmp/oc.tar.gz 
+
+RUN mkdir -p ${APP_ROOT} && touch ${APP_ROOT}/liveness && chmod a+r ${APP_ROOT}/liveness && touch ${APP_ROOT}/readiness && chmod a+r ${APP_ROOT}/readiness
+COPY ./sidecar-logging.sh ${APP_ROOT}
+RUN chmod a+rx ${APP_ROOT}/sidecar-logging.sh
+
+RUN mkdir -p ${APP_LOG}
+RUN chmod 777 ${APP_LOG}
+
+ENTRYPOINT ["/opt/app/sidecar-logging.sh"]
```

Comment:
@sookeke I'm sure you must have tested this in OpenShift but I would recommend following best practices for docker containers in OpenShift and run the container as a **non-root** user: 

```Dockerfile
RUN ...

# Support arbitrary user IDs - OpenShift will use a random UID for docker containers
USER 1001

ENTRYPOINT [...]
```

Source:

* https://developers.redhat.com/blog/2020/10/26/adapting-docker-and-kubernetes-containers-to-run-on-red-hat-openshift-container-platform#runtime_user_compatibility_with_kubernetes
* https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html/creating_images/creating-images-guidelines#openshift-specific-guidelines

Reply:
@asanchezr can you confirm the changes. run container as noon root, use .env file as environment variable etc